packages/website/ts/utils/backend_client.ts

```diff
@@ -45,6 +45,13 @@ export const backendClient = {
     },
     async getTokenInfosAsync(): Promise<WebsiteBackendTokenInfo[]> {
         const result = await fetchUtils.requestAsync(utils.getBackendBaseUrl(), TOKENS_ENDPOINT);
+        // TEMPORARY: Until backend is updated with USDC, this appends the metadata for USDC
+        result.push({
+            address: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+            decimals: 6,
+            name: 'US Dollar Coin',
+            symbol: 'USDC',
+        });
```

Comment:
Does this need to be in here to work, or can we not have this and just add a row to the DB?

Reply:
This is a temporary fix, its better for it to be in the DB, once its in the DB this can be removed

Reply:
Which specific codebase/where is this DB hosted?